PulseSampleApp/PulseSampleApp/Info.plist

```diff
@@ -26,6 +26,11 @@
 	<string>LaunchScreen</string>
 	<key>UIMainStoryboardFile</key>
 	<string>Main</string>
+	<key>NSAppTransportSecurity</key>
```

Comment:
Why are you enabling ArbitraryLoads? Pulse works with https, right? If so you shouldn't need this.